packages/http-server-express/src/http-server-express.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/http-server-express
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as http from 'http';
+import * as https from 'https';
+import * as debugModule from 'debug';
+const debug = debugModule('loopback:http:server:express');
+
+import {
+  BaseHttpContext,
+  BaseHttpHandler,
+  HttpFactory,
+  BaseHttpEndpoint,
+  HttpServerConfig,
+  DefaultHttpEndpoint,
+} from '@loopback/http-server';
+
+import * as express from 'express';
+import {Request, Response, Application as HttpApplication} from 'express';
+
+export {
+  Request,
+  Response,
+  NextFunction,
+  Application as HttpApplication,
+} from 'express';
+
+export {
+  HttpServerConfig,
+  HttpRequestListener,
+  HttpServerLike,
+} from '@loopback/http-server';
+
+export type HttpContext = BaseHttpContext<Request, Response>;
+export type HttpHandler = BaseHttpHandler<Request, Response>;
+export type HttpEndpoint = BaseHttpEndpoint<Request, Response, HttpApplication>;
+
+function toMiddleware(handler: HttpHandler): express.RequestHandler {
+  return (request, response, next) => {
+    debug('Handling request: %s', request.originalUrl);
+    const httpCtx: HttpContext = {
+      req: request,
+      res: response,
+      request,
+      response,
+      next,
```

Comment:
I don't think it's a good idea to expose `next` as a public property of HttpContext that's available to all users. At minimum, we should add some sort of a guard that will throw a helpful error if/when `next` is called multiple times for the same request.